spec/javascript/e2e/SupplementalApplicationPage/preferences.e2e.js

```diff
@@ -7,55 +7,142 @@ import {
   HEADLESS
 } from '../../support/puppeteer/consts'
 
+const getUnselectedOptSelector = (fieldSelector) => {
+  return `${fieldSelector} option:not(:checked):not(:disabled)`
+}
+
+const getSelectedOptSelector = (fieldSelector) => {
+  return `${fieldSelector} option:checked`
+}
+
 describe('SupplementalApplicationPage Confirmed Preferences section', () => {
-  test('should allow preference updates', async () => {
+  test('should allow updates to live/work preference', async () => {
     let browser = await puppeteer.launch({ headless: HEADLESS })
     let page = await browser.newPage()
 
     await sharedSteps.loginAsAgent(page)
 
-    // The application used here must include a claimed and editable
-    // preference as its first preference for this test to be able to pass.
+    // The application used here must include a claimed live/work
+    // preference for this test to be able to pass.
     await sharedSteps.goto(page, `/applications/${LEASE_UP_LISTING_APPLICATION_ID}/supplementals`)
 
-    const firstRowSelector = '.tr-expand:first-child'
-    const firstPanelSelector = `${firstRowSelector} + .tr-expand-content`
-    const statusFieldSelector = `${firstPanelSelector} .form-grid_item:last-child select`
-    const unselectedOptSelector = `${statusFieldSelector} option:not(:checked):not(:disabled)`
-    const selectedOptSelector = `${statusFieldSelector} option:checked`
+    const liveWorkEditSelector = '[id$=in-san-francisco-preference-edit]'
+    const liveWorkRowSelector = '[id$=in-san-francisco-preference-row]'
+    const liveWorkExpandedPanelSelector = '[id$=in-san-francisco-preference-panel]'
 
-    // Click on the first preference's Edit button in the Confirmed
+    // Click on the live/work preference's Edit button in the Confirmed
     // Preferences section to expand that preference's edit panel
-    await page.click(`${firstRowSelector} .button-link`)
+    await page.click(liveWorkEditSelector)
+
+    // Update the individual preference (select Live vs Work in SF)
+    const individualPreferenceSelector = '.individual-preference-select'
```

Comment:
@akegan is there a reason why this selector and the `typeOfProofSelector` are not using `liveWorkExpandedPanelSelector` like the `prefStatusSelector`?